shiny/ui/_input_select.py

```diff
@@ -162,11 +170,17 @@ def input_select(
     ~shiny.ui.input_radio_buttons
     ~shiny.ui.input_checkbox_group
     """
+    if options is not None and selectize is False:
+        raise Exception("Options can only be set when selectize is `True`.")
 
     choices_ = _normalize_choices(choices)
     if selected is None and not multiple:
         selected = _find_first_option(choices_)
 
+    if options is None:
+        options = {}
+
```

Comment:
In R, there's more that happens in the code that handles selectize options: https://github.com/rstudio/shiny/blob/main/R/input-select.R#L199-L231

Should we support that here?